src/install/lockfile.zig

```diff
@@ -3448,23 +3532,23 @@ pub const Package = extern struct {
             };
 
             if (strings.containsChar(input_path, '*')) {
-                if (strings.contains(input_path, "**")) {
-                    log.addError(source, item.loc,
-                        \\TODO multi level globs. For now, try something like "packages/*"
-                    ) catch {};
-                    continue;
-                }
-
-                const without_trailing_slash = strings.withoutTrailingSlash(input_path);
-
-                if (!strings.endsWithComptime(without_trailing_slash, "/*") and !strings.eqlComptime(without_trailing_slash, "*")) {
-                    log.addError(source, item.loc,
-                        \\TODO glob star * in the middle of a path. For now, try something like "packages/*", at the end of the path.
-                    ) catch {};
-                    continue;
-                }
-
-                asterisked_workspace_paths.append(without_trailing_slash) catch unreachable;
+                // if (strings.contains(input_path, "**")) {
+                //     log.addError(source, item.loc,
+                //         \\TODO multi level globs. For now, try something like "packages/*"
+                //     ) catch {};
+                //     continue;
+                // }
+
+                // const without_trailing_slash = strings.withoutTrailingSlash(input_path);
+
+                // if (!strings.endsWithComptime(without_trailing_slash, "/*") and !strings.eqlComptime(without_trailing_slash, "*")) {
+                //     log.addError(source, item.loc,
+                //         \\TODO glob star * in the middle of a path. For now, try something like "packages/*", at the end of the path.
+                //     ) catch {};
+                //     continue;
+                // }
+
+                glob_workspace_paths.append(input_path) catch unreachable;
                 continue;
             } else if (strings.containsAny(input_path, "!{}[]")) {
```

Comment:
can we append to `glob_workspace_paths` for this case too?

Reply:
I think each `input_path` should be appended, because each should be a valid glob pattern